### DISCUSSION TITLE
FroshPlatformThumbnailProcessor results in broken images after deactivating plugin in Shopware 6

### DISCUSSION THREAD

**Discussion by nikkoch** [FriendsOfShopware/FroshPlatformThumbnailProcessor#50] [Q&A]
Anyone knows maybe how to solve the issue? Should we uninstall it completely for images to work?

**Comment by tinect** [comment]
I'd suggest supplying some more information about your problem.
Sadly, "broken" tells nothing :-(

You might miss setting a service?

**Reply by tinect** [reply]
Some questions: 
- which service do you use?
- do the requested urls in storefront look correct (check via devtools)?
- is the thumbnailPattern corretly set?

**Comment by nikkoch** [comment]
hey tinect, thank you for the quick turnaround!

We have deactivated the plugin because it causes layout glitches (adds huge white space under the slider until you drag the browser width or open dev tools) after SW6 update. But after we have deactivated it, cleaned the cache and reindexed media, it still doesn't work. 

Maybe you can have a look here: 

https://stage1.maximex.shop/
https://stage1.maximex.shop/Garten-Freizeit/
https://stage1.maximex.shop/BIO-Blattflecken-Elixier-250-ml/81991500-m

**Reply by tinect** [reply]
When you added media with plugin installed, the media don't get any thumbnail created on disk (=main-feature of the plugin).
So you would have to generate thumbnails with force flag to have them generated after you disabled the plugin. `bin/conole media:generate-thumbnails -strict`

To your layout problems: 
Without seeing the problems live, I don't see any possibility to try to assist you. When you are using custom theme it's hard to try assisting you, too.

**Comment by nikkoch** [comment]
Hey tinect, thank you for the tip. It works for a couple of hours, when we regenerate thumbnails, but images are broken again after several hours. Could you please suggest something? 

Should we deinstall the plugin completely maybe, it is just disabled now? 

The issue why we deactivate the plugin: 
is because there is a huge space added under the slider, when its on. Even via standard theme.

**Reply by tinect** [reply]
For the future: Could you please use "Write a reply" instead of "Suggest an Answer"? This makes the conversion much more understandable for me and others :-)

They got broken after you reactivated the plugin? Or was it active? I can't get the points :-(

The problem with the slider should be fixable by CSS, but I don't know what exactly.
If you have such problems and cannot get the point of errors on your side, you would like not to use the plugin or find anyone trying to make your shop compatible with the plugin OR fixing a possible CSS-Error in plugin.

Besides that: you don't need slider, when you don't have slides. So, you might want to use another element. AND: check pro and contra of sliders in web :-) there are many cons.

I'm really sorry I cannot help you much :-(

**Reply by nikkoch** [reply]
Hello and sorry for the writing in "Suggest an Answer",

It was deactivated , we have regenerated thumbnails and after several hours images were broken again. Nothing was done in the backend. 

This layout glitch is hard to fix via CSS, as it appears/disappears irregularly. 

Also that slide used just for a test as a substitution. Thank you tinect!

**Reply by nikkoch** [reply]
So should deactivating the plugin be enough to have images working or should it be uninstalled completely?

**Reply by tinect** [reply]
Deactivating should be enough.
But: don't keep deactivated plugins in your system. if you don't use them: uninstall and delete them :-)

**Reply by nikkoch** [reply]
we deactivated it, uninstalled and some images still not working after regenerating thumbnails, and clearing the cache. Maybe you have an idea, why it shouldn't work?
Here an example: https://stage1.maximex.shop/Haushalt-Wohnen/Bad-WC/

But once you click on the product and go back, you can see a product image in listing.

**Reply by tinect** [reply]
Were there images generated? There is a message pf generated and skipped. But looking at the shop it lookks like it's working?

**Reply by tinect** [reply]
ah, I see.
400px thumbnail has not been generated for one product.

![image](https://user-images.githubusercontent.com/135993/165989257-5ab3a150-d120-4d74-baa2-bf54051847d5.png)

Did you run the command with the strict option?

**Reply by nikkoch** [reply]
yes, we run it with the --strict option:

"php bin/console media:generate-thumbnails --strict

 // Generating Thumbnails for 14685 files. This may take some time...           

 14685/14685 [============================] 100%

 ----------- -------------------------- 
  Action      Number of Media Entities  
 ----------- -------------------------- 
  Generated   307                       
  Skipped     14378                     
  Errors      0                         
 ----------- -------------------------- "

And sometimes we see all of them regenerated, but after a couple of hours some of them are broken again.
